gnocchi/tests/functional/gabbits/metric-derived.yaml

```diff
@@ -0,0 +1,114 @@
+fixtures:
+    - ConfigFixture
+
+defaults:
+  request_headers:
+    # User foobar
+    authorization: "basic Zm9vYmFyOg=="
+    content-type: application/json
+
+tests:
+    - name: create archive policy
+      desc: for later use
+      POST: /v1/archive_policy
+      request_headers:
+        # User admin
+        authorization: "basic YWRtaW46"
+      data:
+          name: carrot-cake
+          aggregation_methods:
+              - rate:mean
+              - rate:max
+              - rate:95pct
+              - max
+          definition:
+              - granularity: 1 minute
+      status: 201
+
+    - name: create valid metric
+      POST: /v1/metric
+      data:
+          archive_policy_name: carrot-cake
+      status: 201
+
+    - name: push measurements to metric
+      POST: /v1/metric/$RESPONSE['$.id']/measures
+      data:
+          - timestamp: "2015-03-06T14:33:00"
+            value: 10
+          - timestamp: "2015-03-06T14:34:10"
+            value: 13
+          - timestamp: "2015-03-06T14:34:20"
+            value: 13
+          - timestamp: "2015-03-06T14:34:30"
+            value: 15
+          - timestamp: "2015-03-06T14:34:40"
+            value: 18
+          - timestamp: "2015-03-06T14:34:50"
+            value: 20
```

Comment:
just to clarify, when we do rate:mean, how does it compute 14:34:00?

(13-10 + 13-13 + 15-13 + 18-15 + 20-18) / 5?

how does it get the backwindow before the current one if it's not part of the same post?